src/main/java/duchess/command/Command.java

```diff
@@ -0,0 +1,58 @@
+package duchess.command;
+
+import duchess.main.Duchess;
+
+/**
+ * This abstract class contains the logic to handle commands.
+ *
+ * @author Amos Tan
+ * @version CS2103T AY21/22 Semester 1
+ */
+
+public abstract class Command {
+    /** The name of the Command.*/
+    private String name;
+    /** Constructs a Command.*/
```

Comment:
Shouldn't you include @return and @params for your javadocs? Noticed this is missing for other functions also